### ISSUE TITLE
unable to set heatmap options

### ISSUE THREAD

**Issue by Neo-Desktop** [himynameisdave/svelte-frappe-charts#48]
From https://frappe.io/charts/docs/basic/heatmap

The `radius`, `discreteDomains`, and `colors` options are all set at Chart instantiation as opposed to being placed in the data object.

This makes it impossible to customize a heatmap chart

**Comment by mfridman** [comment]
Any chance you could provide an example of how heatmap is working in general (even without the customization)?

Can't seem to render the heatmap so an example would help. (apologies if this is slightly off topic)

**Comment by Neo-Desktop** [comment]
No worries, I don't know if you still need this, but here's a REPL for your consideration
https://svelte.dev/repl/37256533a4044f0c99ade72583dc2883?version=3.42.6

**Comment by gg187on** [comment]
Can I take this issue?

**Comment by himynameisdave** [comment]
@gg187on yes please, I haven't had a lot of bandwidth to work on this project recently.
